src/test/java/com/facebook/openwifirrm/optimizers/tpc/RandomTxPowerInitializerTest.java

```diff
@@ -63,16 +66,32 @@ private static DataModel createModel() {
 		return dataModel;
 	}
 
+	/**
+	 * Return the first value provided by Random, for each key seed
+	 */
+	private static Map<Integer, Integer> mapRngSeedToFirstValue() {
+		Map<Integer, Integer> map = new HashMap<>();
+		map.put(0, 2);
+		map.put(10, 9);
+		map.put(100, 10);
+
+		return map;
+	}
+
 	@Test
 	@Order(1)
-	void testProvidedTxPower() throws Exception {
-		for (int txPower : Arrays.asList(-1, 16, 27)) {
+	void testSeededTxPower() throws Exception {
+		Map<Integer, Integer> seedMap = mapRngSeedToFirstValue();
+
+		for (Map.Entry<Integer, Integer> entry : seedMap.entrySet()) {
```

Comment:
I don't think you need to try to make this test look like what it did before... it's fine to not loop or hardcode any seed -> value pairs, and instead just assert that all devices/radios got the same allocated txPower (whatever that might be).